addons/account_analytic_analysis/account_analytic_analysis.py

```diff
@@ -613,8 +639,8 @@ def fill_remind(key, domain, write_pending=False):
             ]
             base_domain.extend(domain)
 
-            accounts_ids = self.search(cr, uid, base_domain, context=context, order='name asc')
-            accounts = self.browse(cr, uid, accounts_ids, context=context)
+            accounts_ids = self.search(base_domain, order='name asc')
+            accounts = self.browse(accounts_ids)
```

Comment:
No need to rebrowse 'accounts_ids'. Here 'accounts_ids' is already a recordset(s) return by search method !!!